src/Core/Components/Tabs/FluentTabs.razor.cs

```diff
@@ -154,41 +154,44 @@ protected override async Task OnAfterRenderAsync(bool firstRender)
             _jsModuleOverflow = await JSRuntime.InvokeAsync<IJSObjectReference>("import", JAVASCRIPT_FILE.FormatCollocatedUrl(LibraryConfiguration));
 
             var horizontal = Orientation == Orientation.Horizontal;
-            await _jsModuleOverflow.InvokeVoidAsync("fluentOverflowInitialize", _dotNetHelper, Id, horizontal, FLUENT_TAB_TAG,25);
+            await _jsModuleOverflow.InvokeVoidAsync("fluentOverflowInitialize", _dotNetHelper, Id, horizontal, FLUENT_TAB_TAG, 25);
         }
     }
 
     private async Task HandleOnTabChangedAsync(TabChangeEventArgs args)
     {
         var tabId = args?.ActiveId;
-        if (tabId is not null && _tabs.TryGetValue(tabId, out FluentTab? tab))
+        var tab = _tabs.FirstOrDefault(i => i.Id == tabId);
+
+        if (tab is not null && _tabs.Contains(tab))
         {
             await OnTabChange.InvokeAsync(tab);
-            ActiveTabId = tabId;
+            ActiveTabId = tabId!;
```

Comment:
You should test if `tabId` is null before this code. If the compiler informs you that it's possible to have a null value, you should test it.

Reply:
Yes, will add a check (although it can't be null in our situation)